app/api/dao/user.py

```diff
@@ -19,12 +19,12 @@ class UserDAO:
     @staticmethod
     def create_user(data):
         """Creates a new user.
-        
+
```

Comment:
Please remove this formating changes. Thanks

Reply:
These were autogenerated by the editor!

Reply:
I guess it's your lint autosave but OK let's ignore this for now.